frontend-react/src/components/header/ReportStreamNavbar.tsx

```diff
@@ -180,7 +228,7 @@ export const ReportStreamNavbar = ({
             >
                 <div className="usa-nav-container">
                     <div className="usa-navbar">
-                        <Title>
+                        <Title onClick={() => navigate("/")}>
```

Comment:
Do we have to do the routing programmatically? I'm concerned about it breaking the accessibility of a normal link tag.

Reply:
Updated!